content/millennium/dstu2.md

```diff
@@ -534,6 +534,24 @@ The possible `relation` values are:
  `self` | Shows the URL of the current page of results.
  `next` | Shows the URL of the immediate next page of results.
 
+
+### Concurrency
+
+Developers need to account for data concurrency within the response. The FHIR specification states:
+
+>  The results of a search operation are only guaranteed to be current at the moment the operation is executed.
```

Comment:
nit: extra space

discussed a bit offline, we could extract this text to a helper method and call it from both .md files, since they're exactly the same. +1 either way

Reply:
https://github.com/cerner/fhir.cerner.com/pull/312/commits/06d61940fa7813e516325735ea4c01be813c727f#diff-7718c11ed130ac00a9854e0b089f74b4R143